server/stats.go

```diff
@@ -236,6 +251,82 @@ func (s *Server) GetStatSeries(ctx context.Context, in *pb.GetStatSeriesRequest)
 	return &resp, nil
 }
 
+// GetStatAll implements API for Mixer.GetStatAll.
+func (s *Server) GetStatAll(ctx context.Context, in *pb.GetStatAllRequest) (
+	*pb.GetStatAllResponse, error) {
+	places := in.GetPlaces()
+	statVars := in.GetStatVars()
+	if len(places) == 0 || len(statVars) == 0 {
+		return nil, fmt.Errorf("Missing required arguments")
```

Comment:
Could we note which missing argument in the error message?

Reply:
done